path_delete_test.go

```diff
@@ -237,7 +237,7 @@ func TestVersionedKV_Undelete_Put(t *testing.T) {
 	events.expectEvents(t, []expectedEvent{
 		{"kv-v2/data-write", "data/foo"},
 		{"kv-v2/data-write", "data/foo"},
-		{"kv-v2/delete", "delete/foo"},
-		{"kv-v2/undelete", "undelete/foo"},
+		{"kv-v2/delete", "data/foo"},
+		{"kv-v2/undelete", "data/foo"},
```

Comment:
This feels a little off/confusing to me, I left a naming comment on the upstream PR: https://github.com/hashicorp/vault/pull/22487/files#r1302902082